apidef/api_definitions.go

```diff
@@ -539,6 +539,8 @@ type GraphQLConfig struct {
 	ExecutionMode GraphQLExecutionMode `bson:"execution_mode" json:"execution_mode"`
 	// Schema is the GraphQL Schema exposed by the GraphQL API/Upstream/Engine.
 	Schema string `bson:"schema" json:"schema"`
+	// LastSchemaUpdate contains the date and time of the last triggered schema update to the upstream
+	LastSchemaUpdate *time.Time `bson:"last_schema_update" json:"last_schema_update,omitempty"`
```

Comment:
maybe `SchemaUpdatedAt`?

Reply:
Why do you think `SchemaUpdatedAt` is a better choice than `LastSchemaUpdate`?

Reply:
it is common to have fields named like `created_at` and `updated_at`

But as I see we use `LastUpdated` and `DateUpdated`

Reply:
I would say that none of them is a superior choice: `updated_at`, `last_modified`, `last_update` are all common terms you find in the wild so I would suggest to not "overengineer" this term here :-)